src/lib/analyzeOas.ts

```diff
@@ -0,0 +1,213 @@
+    url: 'https://docs.readme.com/main/docs/openapi-extensions#disable-the-api-explorer',
+  },
+  'x-readme.proxy-enabled': {
+    description:
+      "The x-readme.proxy-enabled extension allows you to toggle if API requests from API documentation should be routed through ReadMe's CORS Proxy. You should only need to use this if your API does not support CORS.",
+    url: 'https://docs.readme.com/main/docs/openapi-extensions#cors-proxy-enabled',
+  },
+  'x-readme.samples-languages': {
+    description:
+      'The x-readme.samples-languages extension allows you to toggle what languages are shown by default for code snippets in your API documentation.',
+    url: 'https://docs.readme.com/main/docs/openapi-extensions#code-sample-languages',
+  },
+  'x-readme.samples-enabled': {
+    description:
+      'The x-readme.samples-enabled extension allowed you to disable code samples on specific endpoints. It is no longer supported and can be safely removed from your API definition.',
+    url: 'https://docs.readme.com/main/docs/openapi-extensions#disable-code-examples',
+  },
+  raw_body: {
+    description:
+      "The RAW_BODY property allows you to define that a request body should have its payload delivered as a raw string. This legacy feature is specific to our Manual API editor and we don't recommend you use it outside of that context, instead opting for a traditional `type: string, format: blob` schema definition.",
+  },
+};
+
+/**
+ * Analyze a given OpenAPI or Swagger definition for any OpenAPI, JSON Schema, and ReadMe-specific
+ * feature uses it may contain.
+ *
+ */
+async function analyzeOas(definition: OASDocument) {
+  return analyzer(definition).then((analysis: Analysis) => {
+    if (analysis.openapi) {
+      Object.entries(OPENAPI_FEATURE_DOCS).forEach(([feature, docs]: [keyof Analysis['openapi'], AnalyzedFeature]) => {
+        // eslint-disable-next-line no-param-reassign
+        analysis.openapi[feature] = {
+          ...analysis.openapi[feature],
+          ...docs,
+        };
+      });
+    }
+
+    if (analysis.readme) {
+      Object.entries(README_FEATURE_DOCS).forEach(([feature, docs]: [keyof Analysis['readme'], AnalyzedFeature]) => {
+        // If this ReadMe feature isn't in our resulted analysis result then it's a deprecated
+        // feature that this API definition doesn't contain so we don't need to inform the user of
+        // something they neither use, can't use anyways, nor should know about.
+        if (!(feature in analysis.readme)) {
+          return;
+        }
+
+        // eslint-disable-next-line no-param-reassign
+        analysis.readme[feature] = {
+          ...analysis.readme[feature],
+          ...docs,
+        };
+      });
+    }
+
+    return analysis;
+  });
+}
+
+export function getSupportedFeatures() {
+  return [
+    // OpenAPI features
+    ...Object.keys(OPENAPI_FEATURE_DOCS),
+
+    'readme', // A catch-all for ReadMe features and extensions. Will look for everything.
```

Comment:
Because having people do `--feature=x-readme.explorer-enable` is gnar they can do `--feature=readme` and we'll just query all of their ReadMe extension and feature usage instead. They won't be able to do piecemeal lookups on these but they get a cleaner API instead looking for some of the data they want.